### PR TITLE
Moving Attack simulation api from beta to GA

### PR DESCRIPTION
**Pull request by yadavgaurav121**
This PR contains
- Beta api migration changes to GA : Exact same 
- Two new api in beta and GA
- Updated author to stuartcl

Roadmap # 15188.

### CODE REVIEW

**Comment by yadavgaurav121** [comment]
There is a warning related to [api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/attackSimulation-GA-Changes/api-reference/beta/toc.yml) which is not related to my change

**Comment by yadavgaurav121** [comment]
This PR contains
1. Beta api migration changes to GA
2. Two new api in beta and GA
3. Updated author to stuartcl

**Comment by yadavgaurav121** [comment]
Approved CSDL and API.MD details : https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/11460?path=/reviews/14715-GA-attacksimulation-graphAPI/api.md&_a=files

**Comment by JarbasHorst** [comment]
Hello @yadavgaurav121, I reviewed your PR and left some comments. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [18640](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18640) into your branch to include my fixes and before addressing my comments.

File: api-reference/beta/api/attacksimulationroot-get-simulationautomations.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+### Request
+
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "name": "list_simulationautomation"
+}
+-->
+``` http
```
[the remark this answers is not in the document]
Comment (yadavgaurav121) [reply api-reference/beta/api/attacksimulationroot-get-simulationautomations.md:80]: addressed

File: api-reference/beta/includes/attacksim-deprecate-queryurl-reportapi.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-author: "Gopal-MSFT"
+author: "stuartcl"
 ms.localizationpriority: high
 ms.prod: "reports"
 ms.topic: include
```
[the remark this answers is not in the document]
Comment (yadavgaurav121) [reply api-reference/beta/includes/attacksim-deprecate-queryurl-reportapi.md:5]: Addressed

File: api-reference/v1.0/api/securityreportsroot-getattacksimulationrepeatoffenders.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /reports/security/getAttackSimulationRepeatOffenders
+GET /reports/getAttackSimulationRepeatOffenders
```
[the remark this answers is not in the document]
Comment (yadavgaurav121) [reply api-reference/v1.0/api/securityreportsroot-getattacksimulationrepeatoffenders.md:34]: Updated for : 
1. api-reference/v1.0/api/securityreportsroot-getattacksimulationrepeatoffenders.md
2. api-reference/v1.0/api/securityreportsroot-getattacksimulationsimulationusercoverage.md
3. api-reference/v1.0/api/securityreportsroot-getattacksimulationtrainingusercoverage.md

For api-reference/v1.0/api/simulationautomation-list-runs.md such change is not required

**Comment by yadavgaurav121** [comment]
> Hello @yadavgaurav121, I reviewed your PR and left some comments. Please, take a look when you will have time.
> 
> Also, to avoid merge conflicts, please merge PR [18640](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18640) into your branch to include my fixes and before addressing my comments.

@Jarbas-MSFT Addressed your comments. 
I have have made few changes to your changes as they seems inaccurate for few.

**Comment by JarbasHorst** [comment]
Hi @yadavgaurav121, for clarity, the combination of "isCollection": true and "@odata.type": "microsoft.graph..." is also a valid approach to indicate that the response object is an array.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [18678](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18678) into this branch to include my final edits.

**Comment by yadavgaurav121** [comment]
@Lauragra Please do not merge this once approved. There is a small change is discussion for GA.

**Comment by yadavgaurav121** [comment]
@Lauragra This can be merged now. I have removed the do not merge tag as well.

**Pull request merged**